7 404page

kevinc2465 opened the pull request:
did a simple 404 page, pls tell me if it looks bad

kevinc2465 committed:
drunk

made and add 404 page (needs further editing)

kevinc2465 committed:
done?

bridgetf7 reviewed (changes requested):
- I like the "Return to island" button
- saying the seagull died is potentially too morbid?
- change the bg of the page itself to be transparent so it's not white
- **write in functionality to reroute users to 404 if the page they are trying to go does not exist on the router**
- make seagull smaller by ~30%
thank you!

kevinc2465 committed:
heeheehawhaw

-change the entire bg to transparent, but changed the bg for the div to be white
-changed the phrasing
-made img smaller
-reoute users to 404

bridgetf7 committed:
style changes

- transparent image
- white text
- text size
